tests/testthat/test-derive_param_wbc_abs.R

```diff
@@ -0,0 +1,125 @@
+test_that("Test adding absolute records for each by group", {
+  input <- tibble::tribble(
+    ~USUBJID, ~PARAMCD, ~AVAL, ~PARAM, ~VISIT,
+  "P01",    "WBC",     33,   "Leukocyte Count (10^9/L)",       "CYCLE 1 DAY 1",
+  "P01",    "WBC",     38,   "Leukocyte Count (10^9/L)",       "CYCLE 2 DAY 1",
+  "P01",    "LYMLE",   90,   "Lymphocytes (fraction of 1)", "CYCLE 1 DAY 1",
+  "P01",    "LYMLE",   70,   "Lymphocytes (fraction of 1)", "CYCLE 2 DAY 1"
```

Comment:
AVAL for `"Lymphocytes (fraction of 1)"` should be 0.9, 0.7, etc.

Reply:
Updated.